### PR TITLE
Ported chess toys from Polaris.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
Ports chess props from Polaris, used for map-scale chess games.

image

## Why and what will this PR improve
I'm picking apart another large Polaris feature parity PR and this is part of it. I'm not actually sure if this should go into dev or not. It's fun for mapping but that's about it.

## Authorship
Original is from Polaris, not sure if it originated there.
New icons are from Doe, who is very good.

## Changelog
Nothing player-facing.

### CODE REVIEW

**Review by afterthought2 — COMMENTED** [review]
Possibly "duplicating" `/obj/item/chems/food/checker/pawn` and other pieces.

**Comment by MistakeNot4892** [comment]
Different use-case, these are like, a foot tall. Not chessboard pieces.

**Review by out-of-phaze — DISMISSED** [review]
Looks good, but I have a question: why have separate blue/purple sprites instead of just using the `color` var?

**Comment by MistakeNot4892** [comment]
Because those are the sprites that were on Polaris and this is a minimum effort PR.

Doe is actually redoing them to be isometric and gresycaled - if that goes ahead I'll gank them to replace these.

**Review by out-of-phaze — DISMISSED** [review]
Neat. I like the new sprites even though the knight looks a little scale-y.

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -768,3 +768,95 @@
...
+
+//////////////////////////////////////////////////////
+//					Chess Pieces					//
+//////////////////////////////////////////////////////
+
+/obj/item/toy/chess
+	name = "chess piece"
+	desc = "This should never display."
+	icon = 'icons/obj/items/chess.dmi'
+	w_class = ITEM_SIZE_SMALL
```
Comment (afterthought2) [inline code/game/objects/items/toys.dm]: I don't actually know how obvious it is that these are big and the others are small; there are lots of not-to-scale sprites. I've never seen the others in game so could be off the mark, but would argue that either (a) replace the others or (b) make it more transparent that these are big via item var values and desc.

Comment (out-of-phaze) [reply code/game/objects/items/toys.dm]: Isn't there a long-term goal of making world sprites to-scale? That's why the icon system is the way it is.
That said, updating the w_class value might be good, yes.

**Comment by PlayerDeer** [comment]
> Neat. I like the new sprites even though the knight looks a little scale-y.

Thanks! I do see what you mean... If I can think of a way to make it look better I'll open a new PR for the change! Thanks for the feedback.

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/game/objects/items/toys.dm]: Renamed them to 'oversized'. A more detailed integration can happen in another PR, I just want these for mapping compatibility.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**